### DISCUSSION TITLE
Blog comment on Unbody: Using Weaviate to Build the Foundation for AI-First App Development

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#2713] [Docs/blog comments]
# Blog comment on Unbody: Using Weaviate to Build the Foundation for AI-First App Development

Transforming data scattered across various sources into actionable insights, like searching or building chatbots, involves many complex tasks. Unbody - built on Weavaite’s technology - automates it all with just a few clicks and a line of code.

https://weaviate.io/blog/unbody-weaviate

**Comment by Tabintel** [comment]
Great project with Weaviate and Unbody! ⭐
